src/components/form/HdTextarea.vue

```diff
@@ -171,8 +191,25 @@ export default {
 </script>
 
 <style scoped lang="scss">
+@import 'homeday-blocks/src/styles/mixins.scss';
+
 .textarea {
   display: block;
   resize: vertical;
 }
+.maxlength-indicator {
+  position: absolute;
+  right: 0;
+  bottom: -#{($sp-s * 2) + ($sp-xs / 2)};
```

Comment:
I understand that we have to add a specific amount of spacing from the counter to the text area.

But why do we have those operations?

I'm asking that, because we should only use what we have mapped in the DS without the need to do things like `sp-s + sp-xs`... for example, doing that sum will result in `12px` which is not present in the DS.

If you have to do that to conform to the design spec, we should then talk with the design team to either adapt the spec to what we have in the DS or add a new spacing to it.

Reply:
+1
You don't always have to force the usage of `$sp` values. Unless it's really needed (usually when using absolute/fixed positioning, where you want to add the padding of the container + the margin that you want for your element), but that should stay just for edge cases.
This one looks very complicated and hard to understand too, I would go for creating a variable with a semantic name and give it a hardcoded value for now (e.g. `$distanceFromAtoB: 99px;`), and we can discuss this use case in the DS sync if needed. You can also see if you can combine `bottom` and `margin-bottom` to achieve a more reasonable implementation.

Reply:
I also suggest writing the operations without wrapping them in SASS `#{..}` because if I remember correctly it doesn't work in IE11 😅

Reply:
ℹ️  I have a problem on localhost, will push the changes in a couple of hours.